zephyr/CMakeLists.txt

```diff
@@ -711,6 +711,9 @@ zephyr_library_sources_ifdef(CONFIG_COMP_GOOGLE_HOTWORD_DETECT
 zephyr_library_sources_ifdef(CONFIG_DTS_CODEC
 	${SOF_AUDIO_PATH}/module_adapter/module/dts/dts.c
 )
+if (CONFIG_DTS_CODEC)
```

Comment:
This broke the stub build. Please fix

Reply:
> This broke the stub build. Please fix

@joechengxperi it is the responsibility of the submitter to look at failures, not the reviewers. Ask for help if you don't understand some failures (or worse: if you don't have access to some)

https://github.com/thesofproject/sof/actions/runs/6718130608/job/18257259780?pr=8379

The last error messages can be misleading. Always scroll up and look at the first errors first. Parallel builds may require scrolling quite a way up.

Reply:
Fixed. Apologize that I did not notice that build error.